src/aliases/store/postgres/alias.go

```diff
@@ -73,5 +73,5 @@ func (s *AliasStore) ListAliases(ctx context.Context, registry aliasent.Registry
 }
 
 func (s *AliasStore) DeleteRegistry(ctx context.Context, registry aliasent.RegistryName) error {
-	return goerrors.New("not implemented")
+	return errors.NotImplementedError("DeleteRegistry not implemented")
```

Comment:
As part of the common practice over our code base, we should be logging error happens, EXCEPT when you are in `pkg/` which acts as libraries, and in that case we log on the first component under `/src`